README.md

```
@@ -120,14 +120,60 @@ What would you like to do?
 2) What is the ORDER_ITEMS table?
 3) View Employees under the Employee Tab *** AS OF 6/15/2024 at 1106AM completed ***
 4) One big Inventory Menu with sub menus? (Not being able to view inventory before adding things doesnt make sense)
-5) If there are no deliveries, return (Currently no deliveries)
-6) If there are no orders, return (currently no orders)
-7) When adding an item to inventory, if the price is 13.99, the database can only accept an int (change to float)
-8) Price Not in the Print() when viewing inventory && seperator between Price, Quantity, and ManID in View Inventory
-9) Seperator between ManAddress and DayOfDelivery in View Manufacturers
-10) When creating orders, it says after inputing ManuID (***why manu id isnt this the company's employee?***)
-11) when going back iin Search menu it needs to be go back on 3 not 4
-12) When changing quantity of items in stock, maybe when prompting show how many items are currently in stock in the system? 
+    *** OLD MENU ***
```

Markdownlint (0.37.0)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)